fe/fe-core/src/test/java/org/apache/doris/nereids/stats/FilterEstimationTest.java

```diff
@@ -132,7 +132,7 @@ public void testNotInNaN() {
         Statistics stat = new Statistics(1000, slotToColumnStat);
         FilterEstimation filterEstimation = new FilterEstimation();
         Statistics expected = filterEstimation.estimate(notIn, stat);
-        Assertions.assertTrue(Precision.equals(333.33, expected.getRowCount(), 0.01));
+        Assertions.assertTrue(Precision.equals(666.666, expected.getRowCount(), 0.01));
```

Comment:
I have some questions about this UT
1. why isUnknown=true In line 130
2. we do not modify the algorithm, why case result changed?

Reply:
Calculation logic for `Not` actually  changed